src/Setup/DevDivInsertionFiles/DevDivInsertionFiles.csproj

```diff
@@ -50,6 +50,7 @@
     <ExpectedDependency Include="Humanizer.Core"/>
     <ExpectedDependency Include="ICSharpCode.Decompiler"/>
     <ExpectedDependency Include="Microsoft.DiaSymReader"/>
+    <ExpectedDependency Include="Microsoft.CodeAnalysis.FlowAnalysis.Utilities"/>
```

Comment:
Also, we need to figure out if and how do we want to ngen this new assembly. @tmat Can we use the same model currently used for "System.Reflection.Metadata" if we decide to insert it as part of roslyn?